src/Notes/DataStore.php

```diff
@@ -71,7 +74,7 @@ public function read( &$note ) {
 		if ( 0 !== $note_id || '0' !== $note_id ) {
 			$note_row = $wpdb->get_row(
 				$wpdb->prepare(
-					"SELECT name, type, locale, title, content, icon, content_data, status, source, date_created, date_reminder, is_snoozable FROM {$wpdb->prefix}wc_admin_notes WHERE note_id = %d LIMIT 1",
+					"SELECT name, type, locale, title, content, content_data, status, source, date_created, date_reminder, is_snoozable, layout, image FROM {$wpdb->prefix}wc_admin_notes WHERE note_id = %d  AND is_deleted = 0 LIMIT 1",
```

Comment:
This is causing an error for me - icon is not set in `note_row` but it is needed further below.

Reply:
Thank you for the warning @becdetat.
Are you using the test branch (`add/4099_test_branch`) for testing?

Reply:
Oh, no I wasn't. That's much better.